### PR TITLE
perf(cdk/scrolling): use passive listeners for scroll events

### LINKED ISSUE #24693: feat(ScrollDispatcher): Scroll sdk should use passive event listeners for scroll event
### Feature Description

I found that Lighthouse test failed on "use passive event listeners" check.
Problem is in these lines https://github.com/angular/components/blob/460b25cad996447e6a48175265159d27bb33a133/src/cdk/scrolling/scroll-dispatcher.ts#L185
https://github.com/angular/components/blob/460b25cad996447e6a48175265159d27bb33a133/src/cdk/scrolling/scrollable.ts#L52
image

Using passive listeners will be better for app performance ( https://developer.chrome.com/blog/passive-event-listeners/ )
I think it should be changed to 
`fromEvent(window.document, 'scroll', { passive: true })` and 
`fromEvent(this.elementRef.nativeElement, 'scroll', { passive: true })`

I can create a PR for this if necessary. (Update: already created #24695)

### Use Case

_No response_

### PR DESCRIPTION
**Pull request by vpetrusevici**
Closes #24693

### CODE REVIEW

File: src/cdk/scrolling/scroll-dispatcher.ts [hunk 1]
```diff
@@ -182,7 +182,7 @@ export class ScrollDispatcher implements OnDestroy {
   private _addGlobalListener() {
     this._globalSubscription = this._ngZone.runOutsideAngular(() => {
       const window = this._getWindow();
-      return fromEvent(window.document, 'scroll').subscribe(() => this._scrolled.next());
+      return fromEvent(window.document, 'scroll', { passive: true }).subscribe(() => this._scrolled.next());
```
[the remark this answers is not in the document]
Comment (vpetrusevici) [reply src/cdk/scrolling/scroll-dispatcher.ts:185]: Only in this place or both? (In fact window.document event wasn't my problem, I just added here also to be sure)

[the remark this answers is not in the document]
Comment (vpetrusevici) [reply src/cdk/scrolling/scroll-dispatcher.ts]: Reverted for this place

[the remark this answers is not in the document]
Comment (crisbeto) [reply src/cdk/scrolling/scroll-dispatcher.ts]: It applies to both.

[the remark this answers is not in the document]
Comment (vpetrusevici) [reply src/cdk/scrolling/scroll-dispatcher.ts]: Yep, you are right, this is unnecessary changes, sorry

**Pull request closed without merging**